lib/tugboat/middleware/check_credentials.rb

```diff
@@ -8,7 +8,7 @@ def call(env)
         # We use a harmless API call to check if the authentication will
         # work.
         begin
-          env['barge'].droplet.all.list
+          env['barge'].droplet.all(per_page=1, page=1).list
```

Comment:
This needs to be a hash, resolved in the rebase 7cc77eafbb4d7ef8469b5246bafbb9995f6bf397